nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh

```diff
@@ -15,7 +15,7 @@ GROUP=${DAEMON_GROUP-${6-"daemon"}}
 mkdir -p ${STORAGE_DIR}/logs
 mkdir -p ${STORAGE_DIR}/db
 
-chown -R ${USER}:${GROUP} ${STORAGE_DIR}
+#chown -R ${USER}:${GROUP} ${STORAGE_DIR}
```

Comment:
?

Reply:
There is a problem to chown mouted volume - STORAGE_DIR. Docker can't chown at this path.